examples/solids/src/boundary.c

```diff
@@ -86,15 +86,11 @@ PetscErrorCode BCTwistX(PetscInt dim, PetscReal loadIncrement,
 
   PetscFunctionBeginUser;
 
-  PetscScalar theta = twistMax*M_PI;
+  PetscScalar theta = twistMax*M_PI*loadIncrement;
+  PetscReal c = cos(theta), s = sin(theta);
   u[0] = 0;
-  u[1] = (y*cos(theta) - z*sin(theta))*loadIncrement;
-  u[2] = (y*sin(theta) + z*cos(theta))*loadIncrement;
-
-// Testing
-PetscScalar x = coords[0];
-printf("%f\n", x);
-
+  u[1] = (c - 1.)*y - s*z;
+  u[2] = s*y + (c - 1.)*z;
```

Comment:
@jeremylt Recall that we're setting the displacement of a rotation `(Q - I) x`, not the coordinates of the rotated point `Q x`.

I also changed the load increment to apply the twisting, otherwise you'd see no difference between zero and 2pi rotation, for example.

Reply:
Right. I had something closer ( without the (c - 1) ) but I horribly mangled it while trying to figure out what was going on.